src/st_stuff.c

```diff
@@ -1038,11 +1039,29 @@ void ST_loadGraphics(void)
   {
     sbar  = (patch_t *) W_CacheLumpName("STBAR", PU_STATIC);
     sbarr = NULL;
+
+    if (sbar && sbar->height != ST_HEIGHT)
+    {
+      I_Printf(VB_WARNING, "ST_loadGraphics: Non-standard STBAR height of %d. "
+                           "Expected %d.", sbar->height, ST_HEIGHT);
+    }
   }
   else
   {
     sbar  = (patch_t *) W_CacheLumpName("STMBARL", PU_STATIC);
     sbarr = (patch_t *) W_CacheLumpName("STMBARR", PU_STATIC);
+
+    if (sbar && sbar->height != ST_HEIGHT)
```

Comment:
Not sure if we even need to catch this case. This is strictly only for Doom < 1.2 compatibility.

Reply:
Fixed by https://github.com/fabiangreffrath/woof/pull/1223/commits/4b284237376c28ff4c8364050fda7f342f90efa2.